int32 connect_back_segment
int32 connect_front_segment
uint32[] beam_serials
uint32[] point_serials
int32 nearest_history_serial
float32 nearest_dis
uint8 segment_status
geometry_msgs/Point32 leftPoint
geometry_msgs/Point32 rightPoint
uint8 right_condition
uint8 left_condition
sensing_on_road/line_piece[] line_pieces
uint32[] perpendicular_lines
geometry_msgs/PointStamped segment_centroid_laser
geometry_msgs/PointStamped segment_centroid_odom
float32 centroid_range_to_laser
float32 max_dimension
float32 internal_std
float32 radius
float32 mean_average_deviation
